ossm/merge_upstream.sh

```diff
@@ -49,13 +52,22 @@ merge() {
   return $?
 }
 
-main () {  
+updateVersionsInOssmValuesYaml() {
+    latest_version=$(yq '.versions[0].version' < "${VERSIONS_YAML_FILE}")
```

Comment:
Please double check yq is available on the host where this is running.

Reply:
I _believe_ this is run on `ci/maistra-builder:upstream-master` on the prow cluster, I pulled that image locally and verified that `yq` is there.